build.gradle.kts

```diff
@@ -182,6 +182,12 @@ allprojects {
             testImplementation("org.junit.jupiter:junit-jupiter-params")
             testImplementation("org.hamcrest:hamcrest")
             testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine")
+            // Changes to properties files should be reflected when running any test
+            testRuntimeOnly(files("../build.properties"))
+            testRuntimeOnly(files("../ssltest.properties"))
+            if (file("../build.local.properties").exists()) {
+                testRuntimeOnly(files("../build.local.properties"))
```

Comment:
Please use inputs.file(...) API: https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:task_input_output_runtime_api

Reply:
Can you give an example? I'm not familiar with those APIs.